tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/fullscreen_image.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, marionette):
         Wait(self.marionette).until(expected.element_displayed(
             Wait(self.marionette).until(expected.element_present(
                 *self._current_image_locator))))
+        Wait(self.marionette).until(lambda m: self.marionette.find_element(*self._current_image_locator).get_attribute('src') != '')
```

Comment:
I also changed the Media frame to use a single img element that is always displayed and present, so I added another wait here to wait until the src attribute is set instead of just waiting for the element to be displayed.